psi4/driver/procrouting/proc.py

```diff
@@ -3464,17 +3465,26 @@ def format_vector(label, data):
         lines += [""]
         core.print_out("\n".join(lines) + "\n")
 
+    gauge = core.get_option("ADC", "GAUGE").lower()
     computed = {}
     if any(prop in properties for prop in ("TRANSITION_DIPOLE", "OSCILLATOR_STRENGTH")):
         data = state.transition_dipole_moments
         computed["Transition dipole moment (in a.u.)"] = data
         adc_wfn.set_variable(name + " transition dipoles", core.Matrix.from_array(data))
 
     if "OSCILLATOR_STRENGTH" in properties:
-        data = state.oscillator_strengths.reshape(-1, 1)
-        computed["Oscillator strength (length gauge)"] = data
+        if gauge == "velocity":
+            data = state.oscillator_strengths_velocity.reshape(-1, 1)
+        else:
+            data = state.oscillator_strengths.reshape(-1, 1)
+        computed["Oscillator strength (" + gauge + " gauge)"] = data
```

Comment:
Could you use `f`-strings for these string operations?

Reply:
With pleasure. I thought to leave it out for compatibility to 3.5.